geoh5py/objects/curve.py

```diff
@@ -26,17 +26,74 @@
 from .points import Points
 
 
-class Curve(Points):
+class CellObject(Points):
```

Comment:
**suggestion:** move class to a dedicated file, as it will be shared by other object types

Reply:
Done